en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md

```diff
@@ -0,0 +1,215 @@
+# Configuring Keystores in API-M Analytics
+
+WSO2 products use asymmetric cryptography by default for authentication and data encryption. In asymmetric cryptography, keystores (with key pairs and certificates) are created and stored for the product. Keystore is a repository where private keys and certificates can be stored. It is possible to have multiple keystores so that the keys used for different use cases are kept unique. For more information about keystores and its concepts, see [About Asymmetric Cryptography]({{base_path}}/install-and-setup/setup/security/configuring-keystores/keystore-basics/about-asymetric-cryptography).
```

Comment:
It is possible to have multiple keystores so that the keys used for different use cases are kept unique -> shouldn't this be "It is possible to have multiple keystores so that the keys used for different use cases are kept seperately"

Reply:
Fixed